xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -512,6 +516,74 @@ rebuild_master:
         configure_repositories step, set the <emphasis>add_repositories</emphasis> key to anything else than <emphasis>enabled</emphasis>.</para>
         </sect4>
 
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.submit_request">
+          <title>Submit a Request</title>
+
+          <para>The submit request step is the equivalent of the <emphasis>osc submitrequest</emphasis> command.</para>
+
+          <para>The requirements to run a submit request step are:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>There must be a branch created.</para>
```

Comment:
Not sure what you want to say with this. You mean the `source_package` has to exist?

Reply:
Also changed, and the description of the next requirement.